truss/templates/server/model_wrapper.py

```diff
@@ -90,29 +93,30 @@ def __init__(self, config: Dict):
         self.truss_schema: TrussSchema = None
 
     def load(self) -> bool:
-        if self.ready:
-            return self.ready
+        with self._logger.contextualize(request_id="MODEL_LOAD"):
```

Comment:
I'd suggest either:
* Moving MODEL_LOAD, MODEL_STARTUP all to constants or an enum somwhere
* even better -- passing in a structured object instead. I don't know how this would work with loguru, but you could make some sort of class that has a `lifecycle` field (enum of MODEL_STARTUP, MODEL_LOAD, REQUEST) and an optional `request_id` field that would only be set in the `REQUEST` case

Reply:
I like this. Will add.